test/lsp/protocol_test_corpus.cc

```diff
@@ -246,7 +246,8 @@ TEST_F(ProtocolTest, DoesNotMergeFileChangesAcrossNonDelayableRequests) {
 }
 
 TEST_F(ProtocolTest, NotInitialized) {
-    auto msgs = send(*getDefinition("foo.rb", 12, 24));
+    // Don't use `getDefinition`; it only works post-initialization.
```

Comment:
One downside of LSPConfiguration: It doesn't have a rootURI until after initialization. This is only a problem in the few tests that actually try to do things pre-initialization.